app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class TargetsController < ApiController
+      def create
+        target.save!
+      end
+
+      private
+
+      def target_params
+        params.require(:target).permit(:radius, :latitude, :longitude, :title, :topic_id)
+      end
+
+      def target
+        @target = Target.new(target_params)
```

Comment:
Maybe we can use memoization here?
```suggestion
        @target ||= Target.new(target_params)
```